### PR TITLE
UI Makeover

### LINKED ISSUE #118: Fix compiler warnings
We have a few build warnings with this repo:

```
Compiled with warnings.

./src/components/connect.js
  Line 135:103:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 201:53:   Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 221:51:   Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank

./src/components/send.js
  Line 396:12:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 399:26:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 484:13:  Expected '===' and instead saw '=='                                                                                           eqeqeq

./src/components/settings.js
  Line 3:39:   'Icon' is defined but never used                                                                                              no-unused-vars
  Line 6:7:    'settingsPath' is assigned a value but never used                                                                             no-unused-vars
  Line 75:69:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank

./src/components/permissions.js
  Line 5:10:  'constants' is defined but never used  no-unused-vars

./src/components/main.js
  Line 611:37:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

### LINKED ISSUE #158: Setting to clear app state
We should add a button to settings that logs the user out and removes all data in localStorage

### LINKED ISSUE #167: Incorporate feedback on UI refresh
Need to go through all these: 
image

### LINKED ISSUE #179: "Retry" doesn't retry
## Description 
If connecting times out (because the device is unplugged, for example), and you hit the "Retry" button (because the device has since been plugged back in), then the user is told the password needs to be at least 8 characters.

## Expected Behavior

Pressing "Retry" should retry to connect to the device with existing `deviceID` and `password`.

Screen Shot 2021-11-18 at 6 37 27 PM
Screen Shot 2021-11-18 at 6 37 33 PM

### PR DESCRIPTION
**Pull request by alex-miller-0**
Major overhaul of UI + dark theme!

Closes #158
Closes #118
Closes #167
Closes #179

### CODE REVIEW

**Comment by alex-miller-0** [comment]
Still need to test the pairing screen. Just realized I have not updated that one

**Comment by KevinVitale** [comment]
Suggesting change in: #175

File: src/components/deprecation/deprecatedEthWallet.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import 'antd/dist/antd.dark.css'
+
+class DeprecatedEthWallet extends React.Component {
```
Comment (KevinVitale) [inline src/components/deprecation/deprecatedEthWallet.js:4]: What steps are needed to get this component to render so I can see what it looks like?

File: src/components/ethContracts.js [hunk 2]
```diff
@@ -228,7 +216,14 @@ class EthContracts extends React.Component {
             <Table.Column title='App' dataIndex='app' key='app'/>
             <Table.Column title='Source' dataIndex='website' key='website'
               render={url => (
-                <a href={url} target={"_blank"}>Link</a>
+                <a  className="lattice-a" 
+                    href={url} 
+                    target={"_blank"} 
+                    rel={"noopener noreferrer"}
+                    key={`a-${url}`}
```
Comment (KevinVitale) [inline src/components/ethContracts.js:223]: I couldn't figure out where these contract urls (e.g., `v2_curve`, `v2_aave`) get loaded (explain plz); however, some of them need to be updated:
 - Balancer: https://docs.balancer.fi/v/v1/smart-contracts/addresses
 - Curve: https://curve.readthedocs.io/ref-addresses.html
 - Gnosis Safe: should update pack for v1.3.0? (https://github.com/gnosis/safe-contracts/blob/v1.3.0/CHANGELOG.md)
 - Opyn v2: link out of date, but no better link found...
 - SushiSwap: Oooops — **I lol'd when I found the ADDRESSES & LINK forwards to UniSwap 😂**
 - UniSwap: https://github.com/Uniswap/v3-periphery/blob/main/deploys.md

File: src/components/deprecation/deprecatedEthWallet.js (hunk 1 above)
Comment (KevinVitale) [reply src/components/deprecation/deprecatedEthWallet.js:4]: Remove unnecessary components.

Comment (alex-miller-0) [reply src/components/deprecation/deprecatedEthWallet.js:4]: You can't. Forgot I removed the header switch. I'll remove the component.

**Comment by KevinVitale** [comment]
Additional change suggestion added in: #175

**Review by KevinVitale — CHANGES_REQUESTED** [review]
The new UI looks great! Only things are:
 - removing unnecessary components; and,
 - fixing links to ABI contracts.
 
 There's optional commits in #175 which are only suggestions. All, some, or none are fine with me and don't block this PR in any way.

File: src/components/ethContracts.js (hunk 2 above)
Comment (alex-miller-0) [reply src/components/ethContracts.js:223]: They are loaded from a JSON file stored in a public S3 bucket. Will update it.

Comment (alex-miller-0) [reply src/components/ethContracts.js:235]: 2e23175

File: src/components/deprecation/deprecatedEthWallet.js (hunk 1 above)
Comment (alex-miller-0) [reply src/components/deprecation/deprecatedEthWallet.js]: 2e23175

**Review by KevinVitale — APPROVED** [review]
No more reason to wear my sunglasses while on the site! 😎

**Comment by KevinVitale** [comment]
Observed #179 occurring within this PR.

**Pull request merged**